SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -239,7 +239,9 @@ void FSpatialGDKEditorToolbarModule::AddMenuExtension(FMenuBuilder& Builder)
 		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().StartSpatialDeployment);
 		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().StopSpatialDeployment);
 		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().LaunchInspectorWebPageAction);
+#if PLATFORM_WINDOWS
```

Comment:
So it didn't work :D 
Nice workaround for the time being, can we get a follow-up ticket for this?
I believe we need this working by Dec 6th or something for China...

Reply:
that's the first time I heard that we need this working by Dec 6?

As mentioned in an earlier comment: it is impossible to get this working, we can't build out linux workers on Mac, which is the main reason for this not working.

Reply:
Yeah we're looking to get all GDK services at least compatible by that point. If we can't build on Mac for Linux then we can't deploy though so this is blocked on that regardless. 
Are we looking to add Linux cross compile (does it even exist) or is that not on the roadmap.

Reply:
chatted offline. Unreal doesn't have the functionality to cross compile to linux from Mac, so we can only use Mac for local development